community/sustainability_effort.md

```diff
@@ -0,0 +1,80 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) and in return have their PRs prioritized for review.
+
+Help is also needed for triaging new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels.
+
+## Commitment
+
+The Argo project currently has 4 designated [roles](https://github.com/argoproj/argoproj/blob/main/community/membership.md):
+- Member
+- Reviewer
+- Approver
+- Lead
+
+For those who focus on just one area of the code (such as UI), the `Reviewer` and `Approver` roles can optionally be scoped to just that area.
+
+Any person in a designated role can sign up to participate in this Incentive System.
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member: should focus primarily on authoring PRs, should average a minimum of 8 hours per week; can also assist with triaging bugs or optionally reviewing PRs
+- Reviewer: should author PRs and review PRs to move into the "Approver" role, should average a minimum of 2 hours per week of either PR review time or triaging bugs
+- Approver/Lead: should average a minimum of 2 hours per week of either PR review time or triaging bugs
+
+Note that the hours per week from above is an average over time, and it's fine to have weeks of no activity.
+
+Current roles for individuals can be found in [OWNERS](../OWNERS).
```

Comment:
@agilgur5 are you saying you think there's a problem with not having the members listed anywhere? Is this when we're reviewing the PR to add an individual here and need to verify? I'm not sure what your method was, but it seems to me that the Argo icon shows up in an individual's github profile if they're a member? and one can find a Closed PR in the argoproj repo for their membership as well?

Reply:
Perhaps I can at least clarify that it's reviewers+ that are listed in that document. Let me do that.

Reply:
For posterity, this is a follow-up thread to https://github.com/argoproj/argo-workflows/pull/12359#discussion_r1429416409

Yea I think a clarification would suffice. The term "roles" is defined above as including Members and this line used "roles" while not including Members